In [1]:
print("Veda Technology Internship")
print("Day 1 - Student Marks Analysis")

Veda Technology Internship
Day 1 - Student Marks Analysis


# Student Marks Analysis

## Veda Technology Internship — Day 1

### Objective
The objective of this task is to analyze student marks using Python and calculate the total, average, highest, and lowest marks.

### Tools Used
- Python
- Jupyter Notebook

### Approach
The student marks are stored in a Python list. Built-in Python functions such as `sum()`, `max()`, and `min()` are used to perform the required calculations.

In [2]:
# Student marks dataset

marks = [78, 85, 92, 67, 88, 95, 73, 81, 90, 76]

print("Student Marks:")
print(marks)

Student Marks:
[78, 85, 92, 67, 88, 95, 73, 81, 90, 76]


In [3]:
# Calculate student marks statistics

total = sum(marks)
average = total / len(marks)
highest = max(marks)
lowest = min(marks)

print("Total Marks:", total)
print("Average Marks:", average)
print("Highest Marks:", highest)
print("Lowest Marks:", lowest)

Total Marks: 825
Average Marks: 82.5
Highest Marks: 95
Lowest Marks: 67


## Results

The student marks were successfully analyzed using Python.

- **Total Marks:** 825
- **Average Marks:** 82.5
- **Highest Marks:** 95
- **Lowest Marks:** 67

## Conclusion

The program successfully calculates the total, average, highest, and lowest marks from a list of student marks using basic Python functions. This demonstrates the use of lists and built-in statistical functions for simple data analysis.
